ui/src/components/SignInDialog/index.jsx

```diff
@@ -93,6 +93,7 @@ export default class SignInDialog extends Component {
                 button
                 component="a"
                 href="/login/mozilla-auth0"
+                ref="opener"
```

Comment:
@djmitche This has been depreciated in react 16.3+ , You may need to use 
```js
ref={this.function}
``` 
and define the opener above 😄